automations/data/labels.yml

```diff
@@ -82,6 +82,14 @@ groups:
         description: Bug fix
         emoji: "🛠"
 
+  - name: tooling
+    color: "#362D59"
```

Comment:
Please move this color to the Sentry label instead of the group.

Reply:
right gotcha